index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
```

Comment:
- [ ] [OPTIONAL] This pull request does not have any summary/description, please add a brief detail of what was done on this project. 

![image](https://github.com/Cheptoo21/Hello-Microverse_Project/assets/84156714/1a1fe6b6-6430-4430-8ce0-acfa90495544)